packages/components/addon/components/hds/dropdown/list-item.js

```diff
@@ -86,4 +97,12 @@ export default class HdsDropdownListItemComponent extends Component {
 
     return classes.join(' ');
   }
+
+  @action
+  copyCode() {
+    navigator.clipboard.writeText(this.text);
+    if (navigator.clipboard.readText) {
+      this.isSuccess = true;
+    }
```

Comment:
[issue] a couple of issues: 
- this doesn't work for me in both Safari and Chrome: the text is copied but the style is not "success" (remains the same as before being copied).
- maybe we should compare the value of the `readText` with the value of `this.text` to make sure it's really a "success"? we may have a case in which we have already something in clipboard, but is old and for some reasons the `writeText` failed, but the `readText` doesn't fail because there's already the old value available

Reply:
ok, I'll work on it.